Classes/WPEditorView.m

```diff
@@ -1559,11 +1559,15 @@ - (void)replaceLocalVideoWithID:(NSString *)uniqueID
                    remotePoster:(NSString *)posterURL
                      videoPress:(NSString *)videoPressID
 {
-    NSString * videoPressSafeID = videoPressID;
+    NSString *videoPressSafeID = videoPressID;
     if (!videoPressSafeID) {
         videoPressSafeID = @"";
     }
-    NSString *trigger = [NSString stringWithFormat:@"ZSSEditor.replaceLocalVideoWithRemoteVideo(\"%@\", \"%@\", \"%@\", \"%@\");", uniqueID, videoURL, posterURL, videoPressSafeID];
+    NSString *posterURLSafe = posterURL;
+    if (!posterURLSafe) {
+        posterURLSafe = @"";
```

Comment:
I thought so initially, but this was just the tip of the iceberg. Have a look on this PR https://github.com/wordpress-mobile/WordPress-iOS/pull/4116 for further details.